composer.lock

```diff
@@ -4,20 +4,20 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "e9ed96112baedf6d3f99dd4d2f1bbbbb",
+    "content-hash": "241293f1538a544041bb741bf7fc93b1",
     "packages": [
         {
             "name": "api-platform/core",
-            "version": "v3.2.13",
+            "version": "v3.3.5",
```

Comment:
You need to revert completely the changes on the composer.lock

And then run only `composer update prestashop/hummingbird`